src/initializations.rs

```diff
@@ -74,6 +74,10 @@ pub fn lecun_uniform<T: HasAfEnum>(dims: Dim4) -> Array {
   uniform::<T>(dims, s)
 }
 
+//TODO: permut
+pub fn permut<T: HasAfEnum>(dims: Dim4) -> Array{
+    af::range::<T>(dims, 0).unwrap()
```

Comment:
Did you update your arrayfire backend? `git submodule update --init --recursive` ? The `.unwrap()` calls are not needed anymore.